app/controllers/trainees/confirm_details_controller.rb

```diff
@@ -8,7 +8,11 @@ class ConfirmDetailsController < ApplicationController
     def show
       authorize trainee
       page_tracker.save_as_origin!
-      @confirm_detail = ConfirmDetailForm.new(mark_as_completed: trainee.progress.public_send(trainee_section_key))
+
+      if trainee.draft?
```

Comment:
This mirrors the `if` statement in the `def update` - we don't show the form input for recording progress on non draft trainees. It's starting to feel as though draft and non-draft edits should be dealt with separately... but if so, this should be addressed in the work being undertaken to persist data to the 'confirm' page, not in this PR.

Reply:
Your spidey sense is correct, i feel the same. This can be addressed after that spike work.